conda_build/main_develop.py

```diff
@@ -42,6 +43,21 @@ def main():
                    help=("Relink compiled extension dependencies against "
                          "libraries found in current conda env. "
                          "Do not add source to conda.pth."))
+    p.add_argument(
+                   '-b', '--build_ext',
+                   action='store_true',
+                   help=("Build extensions inplace, invoking: "
+                         "python setup.py build_ext --inplace; "
+                         "add to conda.pth; relink runtime libraries to "
+                         "environment's lib/."))
+    p.add_argument(
+                   '-c', '--clean_build_ext',
```

Comment:
It's redundant, will invoke setup.py clean then exit. And it has different definition of clean than standard conda so was trying to be explicit. But is probably easier to write code to allow clean by itself rather than write custom code to parse. I can update and it is documented clearly